### DISCUSSION TITLE
Apple M2 Support

### DISCUSSION THREAD

**Discussion by nljohnso** [runelite/runelite#15409] [Q&A]
Does Runelite work on the new Apple M2 architecture? I understand it works on M1, but I cannot find anything on M2.

**Comment by jimmydg** [comment]
Apple Silicon supported software for M1 works the same way on M2. I don't have an M2 machine myself so I cannot verify this myself, but I've seen people on YT setting up their M2 laptops with "M1" Apple silicon software.

Have you tried it on your M2? If it works fine, then it's also supported for M2.
